go.mod

```diff
@@ -38,11 +38,11 @@ require (
 	go.uber.org/goleak v1.2.1
 	go.uber.org/zap v1.25.0
 	golang.org/x/crypto v0.12.0
-	golang.org/x/exp v0.0.0-20230713183714-613f0c0eb8a1 // Manually downgraded this to be compatible with libp2p v0.29.2
+	golang.org/x/exp v0.0.0-20230817173708-d852ddb80c63 // Manually downgraded this to be compatible with libp2p v0.29.2
```

Comment:
can remove these manual workarounds for "golang.org/x/exp", just grep for "golang.org/x/exp"